vehicles-utils/upload-insurances/upload-insurances.py

```
@@ -87,17 +200,80 @@ def try_to_map(row):
+def update_vehicle(vehicle_id, data):
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "Authorization": f"Bearer {BEARER_TOKEN}"
+    }
+    params = {}
+    response = requests.put(
+        f"{BASE_URL}/vehicles/{vehicle_id}", data=data, headers=headers, params=params
+    )
+    if response.status_code != 201:
+        logger.info(
```

Ruff (0.8.2)

Undefined name `logger`

(F821)